test/DatePicker.spec.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
```

Comment:
If the test passes with `shallow` let's keep shallow, it's faster :)

Reply:
So we'll need to use both, shallow and mount. Mount to test that it calls XYZ on mount.

Reply:
I have updated this. Did you mean this? :)